src/current/_includes/v23.1/prod-deployment/prod-guidance-cache-max-sql-memory.md

```diff
@@ -1 +1,12 @@
-For production deployments, set `--cache` to `25%` or higher. Avoid setting `--cache` and `--max-sql-memory` to a combined value of more than 75% of a machine's total RAM. Doing so increases the risk of memory-related failures.
\ No newline at end of file
+{% capture formula %}{% include_cached copy-clipboard.html %}<div class="highlight"><pre><code class="language-none" data-lang="none">2 * --max-sql-memory + --cache &lt;= 80% of system RAM
```

Comment:
I think this formula is ambiguous without parens

do you mean

```
(2 * --max-sql-memory) + --cache <= 80% of system RAM
```

or

```
2 * (--max-sql-memory + --cache) <= 80% of system RAM
```

if each were set to 32GB (32000000000 bytes), they differ: 96 GB vs 128 GB

if you type it into most programming languages they do the multiply first due to precedence yielding 96 GB so maybe it's fine

Reply:
@yuzefovich PTAL! This is a good point.